history.py

```diff
@@ -2,3 +2,13 @@
 from ably import AblyRest
 
 client = AblyRest('{{ApiKey}}')
+
+print('Retrieving channel...')
```

Comment:
As with PHP, I don't think we should state retrieving a channel here.  We are instancing a stateless channel so not sure this adds anything but may in fact confuse users by making them think some server-side operation is done when instancing a REST channel.